test/cljam/t_util.clj

```diff
@@ -12,3 +12,15 @@
   (are [?n] (thrown? AssertionError (util/ubyte ?n))
     -1
     256))
+
+(deftest gen-vec
+  (is (= (util/gen-vec 3) [nil nil nil]))
+  (is (= (util/gen-vec 4 1) [1 1 1 1])))
+
+(deftest str->int
+  (is (= (util/str->int "123") 123))
+  (is (= (util/str->int "-456") -456))
+  (is (= (util/str->int "+789") 789))
+  (is (= Integer (type (util/str->int "123"))))
+  (is (= Long (type (util/str->int "12345678901"))))
```

Comment:
Fixed https://github.com/chrovis/cljam/pull/60/commits/74f19e431a2d2418e2c9aef2690b517cb36ae6d3 .

Thanks for your reviews!